tests/py/test_email.py

```diff
@@ -219,6 +220,13 @@ def test_html_escaping(self):
         assert 'foo&#39;bar' in last_email['html']
         assert '&#39;' not in last_email['text']
 
+    @mock.patch('gratipay.models.participant.Participant.send_email')
+    def test_emails_with_plus_are_linked_properly(self, send_email):
+        send_email.return_value = 1
+        self.alice.add_email("foo+bar@example.com")
+        link = send_email.args[1]
+        assert link.startswith('/~alice/emails/verify.html?email=foo%2Bbar%40example.com&nonce=')
+
```

Comment:
@whit537 before merge, we should probably remove this test as well. But I still don't understand why this is passing in the first place. The link now starts with `/~alice/emails/verify.html?email64=`, so the assert *should* fail. Something I don't know about `mock` yet?